gc-check-ins.md

```diff
@@ -0,0 +1,20 @@
+## Regular check-ins with SIG leads
+
+As defined in the [charter of the GC](./governance-charter.md#regular-check-ins-with-sig-leads) the GC will check in with maintainers of all long-term and short-term SIGs on a regular basis. During this check in,
+
+* any project files or roadmap entries should be reviewed to verify that they are still accurate.
+* if a SIG lead has any private concerns they wish to raise with the Governance Committee, this is an opportunity to do so.
+* the health of the SIG will be reviewed, and what measures can be applied to help the SIG.
+
+The check in process is as follows:
+
+* For each SIG a GC member is appointed as a GC liaison. The name of that person will be listed publicly.
+* If a GC member is the maintainer themselves, they are by default the GC liaison for this SIG.
+* The GC liaison is appointed for the duration of the project lifecycle, or until they do not get reelected into the GC. Outside of that regular duration, a GC liaison can resign by providing a named successor.
+* The GC liaison is responsible to check in with the maintainers at least once per month. This can either happen via a (private, non recorded) meeting or through a text-based conversation initiated by the GC liaison.
+* The GC liaison is responsible to bring the concerns of the SIG leads forward to the GC, and they are responsible for ensuring that all action items agreed upon during that check in are logged as issues on the project repository or community repository.
+* The GC liaison will attend the regular project meetings at least once per quarter. If this is difficult to accomplish due to timezone constraints, the GC liaison can be freed from this obligation by the GC. This exception should only rarely be approved.
+* The GC liaison and the maintainers are responsible to provide a quarterly health update, where they assess project health along three dimensions:
+  * Whether the SIG is "On Schedule" relative to its own published roadmap
+  * The "contributor experience": PR turnaround time, newbie-friendly projects, advancement opportunities, automation, etc
```

Comment:
Ideas: # of new contributors, # of new triagers/approvers/maintainers, as well as the churn for those metrics.

Reply:
added